bench/wtperf/wtperf.h

```diff
@@ -67,6 +68,17 @@ typedef struct __truncate_queue_entry TRUNCATE_QUEUE_ENTRY;
 #define ZSTD_BLK BLKCMP_PFX "zstd"
 #define ZSTD_EXT EXT_PFX EXTPATH ZSTD_PATH EXT_SFX
 
+/* Tiered Storage Extensions */
+#ifndef DIR_STORE_PATH
+#define DIR_STORE_PATH "storage_sources/dir_store/libwiredtiger_dir_store.so"
+#endif
+#ifndef S3_PATH
+#define S3_PATH "storage_sources/s3_store/.libs/libwiredtiger_s3_store.so"
+#endif
```

Comment:
@sueloverso , IIRC, the CMake build system does not put shared libraries into the .libs directory. So the S3_PATH should be: storage_sources/s3_store/libwiredtiger_s3_store.so